### PR TITLE
Change StatusLabels to Pills

### PR DESCRIPTION
**Pull request by Droniu**
3.1 port of #1819 

**PR intended to be tested with API branch:**  3.1

### Pull Request Checklist

1. [ ] This code contains UI changes
2. [ ] All visible strings are translated with proper context including data-formatting
3. [ ] Attributes `[data-test-id]` are added for new elements
4. [ ] Changes are mentioned in the changelog
5. [ ] The changes are tested in different browsers and in light/dark mode

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-4618 Dashboard: No indication of interactive components](https://app.clickup.com/t/2549495/SALEOR-4618)

File: src/orders/components/OrderPayment/messages.ts [hunk 1]
```diff
@@ -1,14 +1,99 @@
 import { defineMessages } from "react-intl";
 
-const messages = defineMessages({
+export const orderPaymentMessages = defineMessages({
   orderPaymentClickAndCollectShippingMethod: {
```
Comment (jwm0) [inline src/orders/components/OrderPayment/messages.ts:4]:
```suggestion
  clickAndCollectShippingMethod: {
```

File: src/orders/components/OrderPayment/messages.ts [hunk 2]
```diff
@@ -1,14 +1,99 @@
...
     defaultMessage: "click&collect",
     description: "OrderPayment click&collect shipping method"
   },
   orderPaymentShippingDoesNotApply: {
```
Comment (jwm0) [inline src/orders/components/OrderPayment/messages.ts:8]:
```suggestion
  shippingDoesNotApply: {
```

**Review by Kjaroszew — APPROVED** [review]
OK

**Review by jwm0 — APPROVED** [review]

**Pull request merged**